<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Account</class>
 <widget class="QWidget" name="Account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_Post">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>28</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>头像</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_account">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>账号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_DeleteAvatar">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton_DeleteAvatar {
	
	image: url(:/images/deleteAvatar.png);
    border: 1px solid gray;
    border: none; /* 去掉边框 */
    border-radius: 15px; /* 设置为半径的一半 */
    min-width: 30px;  /* 设置最小宽度 */
    min-height: 30px; /* 设置最小高度 */
    max-width: 30px;  /* 设置最大宽度 */
    max-height: 30px; /* 设置最大高度 */
    text-align: center; /* 使文本居中 */
}
QPushButton#pushButton_DeleteAvatar:hover {
    background-color: lightgreen; /* 悬停时背景色变为浅绿色 */
}
QPushButton#pushButton_DeleteAvatar:pressed {
    background-color: red; /* Background color when pressed */
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
